docs/contract.md

```diff
@@ -0,0 +1,45 @@
+# Our Contract
+
+We revised our contracts early in 2018 to include more specific language about what's expected from both sides,
+such as clearer milestones and how quickly we should provide feedback on updates to your content.
+This page is a simplified human-readable summary of its terms,
+and is not a substitute for the full agreement.
+
+DataCamp's contract with authors is broken into two parts to simplify future agreements:
+a *master agreement* governing our relationship,
+which you only need to sign once,
+and a *content exhibit* with project-specific details.
+This is a shorter document,
+and you will need to sign one copy for each work you do with us
+(not least because it includes delivery dates).
+
+The contract's key features are:
```

Comment:
We probably ought to mention how they get paid.

Reply:
Er. Yes. How *do* they get paid?

Reply:
Once a quarter is as much as I know?

Reply:
added